resources/js/beatmap-discussions/nominations.tsx

```diff
@@ -634,12 +634,36 @@ export class Nominations extends React.PureComponent<Props> {
             : formatDate(rankingEta)
           : trans('beatmaps.nominations.rank_estimate.soon');
 
+        const unresolvedProblems = this.props.beatmapset.discussions.filter((d) => !d.resolved && ['problem', 'suggestion'].includes(d.message_type));
+        const rankEstimatePrefixText = unresolvedProblems.length > 0
+          ? trans('beatmaps.nominations.rank_estimate.unresolved_problems')
+          : trans('beatmaps.nominations.rank_estimate.no_problems');
+
+        const problems = unresolvedProblems.map((problem) => (
+          <a
+            key={problem.id}
+            href={makeUrl({ discussion: problem })}
+            rel='noreferrer'
+            target='_blank'
+          >
+            #{problem.id}
+          </a>
```

Comment:
Use the same format as https://github.com/ppy/osu-web/blob/a7f53f734fc847a924611ebf6b964863122a2362/resources/js/beatmap-discussions/nominations.tsx#L261
It's strange if this link behaves differently